meditation_app/dom.js

```diff
@@ -0,0 +1,87 @@
+const app = () => {
+    const music = document.querySelector('.music');
+    const play = document.querySelector('.play');
+    const outline = document.querySelector('.moving-outline circle');
+    const video = document.getElementById('video');
+    const replay = document.querySelector('.replay');
+
+    // Types of sounds
+    const sounds = document.querySelectorAll('.weather');
+    // Timer
+    const timer = document.querySelector('.timer');
+    // Moving-outline length
+    const outlineLength = outline.getTotalLength();
+    // Time selector
+    const timeSelector = document.querySelectorAll('.time-selector__item');
+    // Time duration
+    let maxDuration = 600;
+
+    outline.style.strokeDasharray = outlineLength;
+    outline.style.strokeDashoffset = outlineLength;
+
+    // Change music
+    sounds.forEach(sound => {
+        sound.addEventListener('click', () => {
+            music.src = sound.getAttribute("data-sound");
+            video.src = sound.getAttribute("data-video");
+            checkPlaying(music);
+        });
+    });
+
+    // Play music
+    play.addEventListener('click', () => {
+        checkPlaying(music);
+    });
+
+    // Replay music
+    replay.addEventListener('click', () => {
+        music.currentTime = 0;
+        video.currentTime = 0;
+    });
+
+    // Change duration 
+    timeSelector.forEach(Option => {
+        Option.addEventListener('click', function() {
```

Comment:
why do you use arrow function for callback in forEach, and functional declaration for callback in event listener?  Can they be the same style, if yes make them consistent? if not what is the difference beetween them?